tools/remark-cli/package-lock.json

```diff
@@ -144,6 +143,7 @@
       "requires": {
         "anymatch": "1.3.2",
         "async-each": "1.0.1",
+        "fsevents": "1.1.3",
```

Comment:
This is an unrelated change as far as I can tell which cascades down to a whole lot of unrelated changes below.

Reply:
It seems like `fsevents` is a optional dependency for `chokidar`. I guess it depends what version of npm you use if this gets added or not.

Reply:
It's required only for very specific OS X versions and breaks installs on Windows. It should not be included in the package-lock.

Reply:
Ok, in that case it should indeed be removed